### PR TITLE
Add label-studio-sdk

### PR DESCRIPTION
**Pull request by CharalamposGeorgiadis**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by CharalamposGeorgiadis** [comment]
@conda-forge/help-python

File: recipes/label-studio-sdk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - python >=3.6
+    - lxml >=4.2.5
+    - requests >=2.22.0,<3
+    - pydantic <3,>1.7
+    - label-studio-tools >=0.0.1
+
+test:
+  imports:
+    - docs
```
Comment (xhochy) [inline recipes/label-studio-sdk/meta.yaml:30]: This looks like the docs are getting installed. You should patch the find_packages call in setup.py/pyproject.toml to omit it.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/label-studio-sdk/meta.yaml (hunk 1 above)
Comment (CharalamposGeorgiadis) [reply recipes/label-studio-sdk/meta.yaml:30]: I modified `setuptools.find_packages`, which was merged yesterday:

` packages=setuptools.find_packages(exclude=["docs"])`

Is this enough/correct? Do I need to wait for the pypi version to be updated before regenerating the recipe?

**Comment by CharalamposGeorgiadis** [comment]
@conda-forge/help-python

I think that this can be merged now

**Review by xhochy — APPROVED** [review]
Thanks!

**Pull request merged**